be/src/olap/storage_engine.cpp

```diff
@@ -468,7 +469,7 @@ OLAPStatus StorageEngine::clear() {
     }
     _store_map.clear();
 
-    delete _memtable_flush_executor;
+    SAFE_DELETE(_memtable_flush_executor);
```

Comment:
```suggestion
    delete _memtable_flush_executor;
```
It's safe to `delete nullptr`, so there is no need to check if it is nullptr.

Reply:
It is safe, SAFE_DELETE is historical problem